+sw_tests/+unit_tests/unittest_spinw_field.m

```diff
@@ -0,0 +1,59 @@
+classdef unittest_spinw_field < sw_tests.unit_tests.unittest_super
+    methods (Test)
+        function test_incorrect_shape_field_raises_error(testCase)
+            swobj = spinw();
```

Comment:
If you want a new object for each test it is possible to define function inside 
`methods (TestMethodSetup)`
as opposed to 
`methods (TestClassSetup)`
which does it once for the whole test suite.
In this case I';m not sure it saves you any lines though!